bitnami/odoo/Chart.yaml

```
@@ -27,4 +27,4 @@ name: odoo
 sources:
   - https://github.com/bitnami/bitnami-docker-odoo
   - https://www.odoo.com/
-version: 18.0.0
+version: 18.0.1
```

```suggestion
version: 18.1.0
```
In this case, more than a fix is a new feature, so I would bump the minor version instead of the patch